### PR TITLE
Tests inputs/outputs folders relative to project test files

### PR DESCRIPTION
**Pull request by ccjoel**
Removed ugly OS-driven manual `sep` (separator) variable. Added ability to run tests from root dir (or elsewhere), since input/output files are now looked relative from elwood_unit_test.py file.

To test this, I rebuilt the docker image and ran the tests both from root-project-dir and under the tests/ folder.

### CODE REVIEW

File: tests/elwood_unit_test.py [hunk 1]
```diff
@@ -37,32 +47,34 @@ def tearDown(self):
         gc.collect()
         # Delete any output parquet files.
         try:
-            os.remove(f"outputs{sep}unittests.parquet.gzip")
-        except:
+            os.remove(output_path("unittests.parquet.gzip"))
+        except FileNotFoundError as e:
```
Comment (ccjoel) [inline tests/elwood_unit_test.py:48]: Ah, I had added this to handle the expected exception (file not found) vs a catch all exception. Not greatly relevant, since this is under unit tests, but not a bad idea regardless.

File: elwood/elwood.py [hunk 2]
```diff
@@ -315,7 +315,7 @@ def load_gadm3(self):
 def optimize_df_types(df: pd.DataFrame):
     """
     Pandas will upcast essentially everything. This will use the built-in
-    Pandas function to_numeeric to downcast dataframe series to types that use
+    Pandas function to_numeric to downcast dataframe series to types that use
```
Comment (ccjoel) [inline elwood/elwood.py:318]: Drive-by typo, heh.

**Review by Sorrento110 — APPROVED** [review]
Looks great to me!
